src/MoBi.Core/Services/ForbiddenNamesRetriever.cs

```diff
@@ -45,12 +47,14 @@ public class ForbiddenNamesRetriever : IForbiddenNamesRetriever,
       IVisitor<IModelCoreSimulation>
 
    {
-      private readonly IMoBiContext _context;
+      private readonly IBuildingBlockRepository _buildingBlockRepository;
+      private readonly ISimulationRepository _simulationRepository;
       private HashSet<string> _nameHashForVisitor;
 
-      public ForbiddenNamesRetriever(IMoBiContext context)
+      public ForbiddenNamesRetriever(IBuildingBlockRepository buildingBlockRepository, ISimulationRepository simulationRepository)
```

Comment:
There are a lot of changes like this - removing IMoBiContext, and using IBuildingBlockRepository instead.

Sometimes IMoBiContext was used for another purpose, so couldn't be removed